spellcheck.yml

```diff
@@ -17,3 +17,10 @@ matrix:
       ignores:
       - code
       - pre
+  - pyspelling.filters.context:
+      context_visible_first: true
+      delimiters:
+      - open: '{{<'
+        close: '>}}'
+      - open: '{{%'
+        close: '%}}'
```

Comment:
Btw, I have no idea if this is actually required, I just copied it over from #3406.

Reply:
FYI @chalin 

This section is just to avoid spellcheck for texts inside shortcodes. (if we don't have this configuration, youtube id will be assumed as typo) 

:)